### PR TITLE
Adding packages for ce88, updating cython version.

### PR DESCRIPTION
**Pull request by gunjanbaid**
@samyag1 @eickenberg Is version 0.25.2 of cython ok for pycortex? Previously, we were using version 0.26 of cython, but another course will need to use 0.25.2. Thanks!

### CODE REVIEW

**Comment by gunjanbaid** [comment]
@yuvipanda @ryanlovett Do these changes look ok? Will wait to merge until you have approved and cogneuro instructors have confirmed that the cython version change is ok.

File: user-image/connectors/neuro.bash [hunk 1]
```diff
@@ -5,7 +5,8 @@ set -euo pipefail
 jupyter serverextension enable  --sys-prefix --py nbserverproxy
 
 # pycortex seems to need cython, but isn't explicitly defined as a dependency
-pip install --no-cache-dir cython==0.26
+# pip install --no-cache-dir cython==0.26
+# cython 0.25.2 installed in Dockerfile		--Gunjan
```
Comment (yuvipanda) [inline user-image/connectors/neuro.bash]: Don't leave in commented out code, remove it. And add a comment in where cython is being installed that it's used for multiple classes.

**Comment by yuvipanda** [comment]
One comment, but otherwise if it builds + testing on staging looks good I'm ok. Thank you for pinging me!

**Comment by ryanlovett** [comment]
Lgtm. It is totally beyond the scope of this issue, but you might consider sending a patch to (or filing an issue with) pycortex to add cython as an explicit dependency.

**Comment by gunjanbaid** [comment]
Removed the commented code, as requested. Also got confirmation from cogneuro instructors that version 0.25.2 of cython is ok to use. If build succeeds, will merge this into staging. Thank you!

**Pull request merged**